### ISSUE TITLE
Horsearmor

### ISSUE THREAD

**Issue by Xillian112** [EasyMFnE/DeadHorses#1]
They DeadHoreses cant wear Horsearmors .. =(

**Comment by EasyMFnE** [comment]
I am pretty sure it's a limitation within Minecraft/Bukkit itself, but I will investigate further and see what I can do.  Even if it isn't possible to equip armor via Inventory gui, it may be possible to equip armor through code.

I'll start my looking with `org.bukkit.inventory.HorseInventory.setArmor(ItemStack)`.

**Comment by EasyMFnE** [comment]
I just did some experiments while I was cooking dinner.  Here's what I found out:
- The two dead horse variants have an armor slot, though it cannot be seen or used normally.
- InventoryClickEvents for that slot can be caught and used to set the slot's item.
- Dead horses equipped with armor:
  - Visually appear to NOT be equipped with armor
  - Take the same damage as unarmored horses

![screenshot from 2014-05-21 18 22 40](https://cloud.githubusercontent.com/assets/3160455/3047972/2c282f5e-e13e-11e3-998c-f8dab00defd8.png)

The damage issue could be worked around by catching the EntityDamageEvent/EntityDamageByEntityEvent and modifying the damage based upon the armor type.  However, I can't imagine a way around the armor not being rendered.

As a result, I am closing this issue.  If, in the future, there would somehow be a change to Minecraft that would allow the functionality, I would be opening to reopening it.
